packages/sanity/src/core/field/types.ts

```diff
@@ -25,40 +25,21 @@ import {
 } from '@sanity/types'
 import {type ComponentType} from 'react'
 
+import {type DocumentGroupEvent} from '../store/events/types'
 import {type FieldValueError} from './validation'
 
-/**
```

Comment:
This is replaced by the event types, that will be part of the `event.chunk.type`